src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -113,6 +114,12 @@ public void parseCommand_addApp() throws Exception {
         assertEquals(new AddAppCommand(appointment), command);
     }
 
+    @Test
+    public void parseCommand_switchiew() throws Exception {
+        assertTrue(parser.parseCommand(SwitchViewCommand.COMMAND_WORD) instanceof SwitchViewCommand);
+        assertTrue(parser.parseCommand(SwitchViewCommand.COMMAND_WORD + " 3") instanceof SwitchViewCommand);
```

Comment:
Does this mean that switchView 3 works?

Reply:
Yes it does! This works similarly for the default Help and Exit commands as well :)